src/zjs_grove_lcd.c

```diff
@@ -255,88 +259,68 @@ jerry_value_t zjs_grove_lcd_init()
     jerry_value_t glcd_obj = jerry_create_object();
     zjs_obj_add_function(glcd_obj, zjs_glcd_init, "init");
 
-    // create object properties
-    jerry_value_t val;
-
     // function flags
-    val = jerry_create_number(GLCD_FS_8BIT_MODE);
+    ZVAL_MUTABLE val = jerry_create_number(GLCD_FS_8BIT_MODE);
```

Comment:
No, this leaks all those values. This is why you shouldn't use MUTABLE. They all still have to be released manually.